ui/components/app/asset-list/asset-list.js

```diff
@@ -69,6 +69,13 @@ const AssetList = ({ onClickAsset }) => {
   return (
     <>
       {process.env.MULTICHAIN ? <BalanceOverview /> : null}
+      {detectedTokens.length > 0 &&
+      !isTokenDetectionInactiveOnNonMainnetSupportedNetwork ? (
```

Comment:
Nit: this and line 71 could use `&&` instead of `? : null`

Reply:
Updated code relevant to this change, but leaving multichain as-is to keep changes minimal.